grafana/resource_team_external_group_test.go

```diff
@@ -0,0 +1,62 @@
+package grafana
+
+import (
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccTeam_External_Groups(t *testing.T) {
```

Comment:
```suggestion
func TestAccTeamExternalGroups(t *testing.T) {
```

Minor nit, but I don't think the underscores are necessary.